### ISSUE TITLE
aString.localeCompare is not a function

### ISSUE THREAD

**Issue by trevorr** [snowcoders/sortier-vscode#110]
I get this error when running on any file with VS Code 1.33.0 on Mac. I'm not sure how to get more information, such as a stack trace.

**Comment by k2snowman69** [comment]
If possible, can you provide a short source code example that it reproduces on along with the file extension? For example:

```
// Blah.js
let aString = "";
aString.localeCompare("");
```

**Comment by trevorr** [comment]
I think it's caused by trying to sort object literals that have numbers as keys. This code reproduces it:

```js
const obj = {
  1: 'a',
  2: 'b'
};
```

**Comment by k2snowman69** [comment]
I don't believe we have any tests for this specific use case so it's possible. I've opened a bug on `sortier` itself to add a unit test for this and investigate. I'm under a tight ship schedule going out next week but should have time after.

https://github.com/snowcoders/sortier/issues/390

If you're interested in investigating yourself, you're welcome to open a PR on Sortier and then I can publish the packages. For javascript files we use [Facebook Flow's parser](https://flow.org/en/docs/config/options/). I'm going to use https://github.com/snowcoders/sortier/issues/390 to track the fixing of the functionality of this issue. After it's fixed there, this issue will track the update of sortier in the vscode package to the latest and publishing a new vs code package.

**Comment by k2snowman69** [comment]
Pull request created
https://github.com/snowcoders/sortier/pull/392

Check out the tests and let me know what you think

**Comment by k2snowman69** [comment]
Just FYI, I just released @snowcoders/sortier@2.5.4 which fixes the issue in the engine and I'm waiting to close out https://github.com/snowcoders/sortier-vscode/pull/116 before publishing a new vscode version.

**Comment by k2snowman69** [comment]
Just released with 1.2.2
